student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Student < Person
+  def initialize(classroom)
```

Comment:
- Great Job for extends parent's constructor by adding classroom however you forget to add the parameters for it so please add the paremeter to fit the requirment 👍
```
`def initialize(age, classroom, name = "Unknown", parent_permission = true)`
```